src/Mapster/Adapters/ClassAdapter.cs

```diff
@@ -23,10 +23,11 @@ protected override bool CanMap(Type sourceType, Type destinationType, MapType ma
             if (sourceType == typeof(string) || sourceType == typeof(object))
                 return false;
 
-            if (!destinationType.IsPoco())
+            if (destinationType.GetTypeInfo().IsEnum || destinationType.Namespace.StartsWith("System"))
                 return false;
 
-            return true;
+            return (destinationType.GetPublicFieldsAndProperties(allowNoSetter: false).Any() ||
+                    destinationType.GetPrivateFieldsAndProperties(allowNoSetter: false).Any());
```

Comment:
sure, I'll create a new class inheriting from ClassAdapter. I thought about it before, but since I'm not too sure about usage of the score in each adapter class (as I understand, it's to determine precedence of which adapter to check first, but then the magic number for each score might have some story behind it, so I didn't want break something). Which score should I give to this new class? 
Also, is there any specific behavior that you are afraid that might break existing logic if we put the code like above? All the unit tests are passing, so at least not something that the library is covering I guess?